ansible_collections/arista/avd/examples/single-dc-l3ls/group_vars/FABRIC.yml

```diff
@@ -49,3 +49,41 @@ bgp_peer_groups:
 # If you're running vEOS-lab or cEOS, you should use MTU of 1500 instead as shown in the following line
 # p2p_uplinks_mtu: 1500
 p2p_uplinks_mtu: 1500
+
+# Set default uplink, downlink, and MLAG interfaces based on node type
+default_interfaces:
+  - types: [ spine ]
+    platforms: [ default ]
+    uplink_interfaces: [ Ethernet1-2 ]
+    downlink_interfaces: [ Ethernet1-8 ]
+  - types: [ l3leaf ]
+    platforms: [ default ]
+    uplink_interfaces: [ Ethernet1-2 ]
+    mlag_interfaces: [ Ethernet3-4 ]
+    downlink_interfaces: [ Ethernet8 ]
+  - types: [ l2leaf ]
+    platforms: [ default ]
+    uplink_interfaces: [ Ethernet1-2 ]
+
+# CloudVision Portal definitions
+cvp_instance_ip: 192.168.1.12
+terminattr_smashexcludes: "ale,flexCounter,hardware,kni,pulse,strata"
+terminattr_ingestexclude: "/Sysdb/cell/1/agent,/Sysdb/cell/2/agent"
+terminattr_disable_aaa: true
+custom_structured_configuration_daemon_terminattr:
```

Comment:
This will be solved by #2800. Please review that so we can get it merged. Then you can remove this.

Reply:
Fixed